pulumi/src/config.ts

```diff
@@ -56,9 +57,21 @@ export const getConfig = async (coinstack: string): Promise<Config> => {
   if (config.statefulService) {
     if (config.statefulService?.replicas === undefined) missingRequiredConfig.push('statefulService.replicas')
 
+    if (config.statefulService.backup) {
```

Comment:
this is the backup config validation I was referring to. pulumi config is not type safe, so we do extra validation to ensure it conforms to our `Config` type as we expect